include/mfmg/exceptions.hpp

```diff
@@ -77,6 +77,7 @@ class NotImplementedExc : public std::exception
 
 inline void ASSERT_THROW_NOT_IMPLEMENTED()
 {
+  std::cerr << boost::stacktrace::stacktrace();
```

Comment:
If you want this you should guard it with `MFMG_WITH_STACKTRACE` otherwise the code won't compile when we don't have stacktrace.